docs/appendices/release-notes/5.8.0.rst

```diff
@@ -45,15 +45,21 @@ Released on 2024-07-17.
 Breaking Changes
 ================
 
+.. _version_5.8.0_break_ch_remove_warmer_enabled:
```

Comment:
Those anchors are added, so that we have a unified layout for the 3 bullet points in the list.
(When added the one for the `number_of_routing_shards` it added a bit more spacing between that and the previous bullet point and rendered a bit ugly.